frontend/tests/integration/components/password-strength-meter-test.js

```diff
@@ -3,7 +3,6 @@ import { setupRenderingTest } from "ember-qunit";
 import { render } from "@ember/test-helpers";
 import { hbs } from "ember-cli-htmlbars";
 import { setLocale } from "ember-intl/test-support";
-
```

Comment:
Make sure to check your pull requests for random unintended leftovers, I don't think deleting that line here makes sense.